vesc_driver:
  port: /dev/ttyACM0

racecar:
  #degrees
  steering_angle_offset: 28.6479
  steering_range_range: 20